_includes/sidebar-data-v2.1.json

```diff
@@ -156,6 +156,12 @@
                 "urls": [
                   "/${VERSION}/demo-json-support.html"
                 ]
+              },
+              {
+                "title": "Performance Benchmarking",
```

Comment:
I think we may want to add this to the Deploy section instead of here, maybe just after "Monitoring and Alerting"?